tests/test_fetch_im_resources.py

```diff
@@ -0,0 +1,262 @@
+                "id": 1,
+                "html_content": qb_question1_content,
+                "answers": [
+                    {
+                        "id": 11,
+                        "html_content": qb_question1_answer1_content
+                    },
+                    {
+                        "id": 12,
+                        "html_content": qb_question1_answer2_content
+                    }
+                ]
+            },
+            {
+                "id": 2,
+                "html_content": qb_question2_content
+            },
+            {
+                "id": 3,
+                "html_content": qb_question3_content
+            }
+        ]
+    )
+
+    output_path = tmp_path / "im_resources"
+
+    mocker.patch(
+        "sys.argv",
+        ["", f"{tmp_path}", f"{output_path}", "mbz"]
+    )
+
+    fetch_im_resources.main()
+
+    assert(len(os.listdir(output_path)) == 3)
+    for filename in os.listdir(output_path):
+        f = os.path.join(output_path, filename)
+        with open(f, 'r') as file:
+            assert file.read() == resource_content.decode()
+
+
+def test_fetch_im_resource_none(
+    tmp_path, mbz_builder, lesson_builder, mocker
+):
+
+    lesson1_page1_content = "<div><p>Lesson 1 Page 1</p></div>"
+    lesson1 = lesson_builder(
+        id=1,
+        name="Lesson 1",
+        pages=[
+            {
+                "id": 11,
+                "title": "Lesson 1 Page 1",
+                "html_content": lesson1_page1_content
+            }
+        ]
+    )
+    mbz_builder(
+        tmp_path,
+        activities=[lesson1]
+        )
+
+    output_path = tmp_path / "im_resources"
+
+    mocker.patch(
+        "sys.argv",
+        ["", f"{tmp_path}", f"{output_path}", "mbz"]
+    )
+
+    fetch_im_resources.main()
+
+    assert (len(os.listdir(output_path)) == 0)
+
+
+def test_fetch_im_resource_repeats(
+    tmp_path, mbz_builder, lesson_builder, mocker, requests_mock
+):
+    filename = "abcd.mp4"
+    im_resource = f"https://s3.amazonaws.com/im-ims-export/{filename}"
+    lesson1_page1_content = f'''
+        <div><p>Lesson 1 Page 1</p>
+        <img src="{im_resource}"></img>
+        <img src="{im_resource}"></img>
+        <img src="{im_resource}"></img></div>'''
+    resource_content = b"123456789abcdef"
+    lesson1 = lesson_builder(
+        id=1,
+        name="Lesson 1",
+        pages=[
+            {
+                "id": 11,
+                "title": "Lesson 1 Page 1",
+                "html_content": lesson1_page1_content
+            }
+        ]
+    )
+    mbz_builder(
+        tmp_path,
+        activities=[lesson1]
+        )
+    m = requests_mock.get(
+        f"https://s3.amazonaws.com/im-ims-export/{filename}",
+        content=resource_content
+    )
+
+    output_path = tmp_path / "im_resources"
+    mocker.patch(
+        "sys.argv",
+        ["", f"{tmp_path}", f"{output_path}", "mbz"]
+    )
+
+    fetch_im_resources.main()
+
+    assert (m.call_count == 1)
```

Comment:
Minor: It looks you don't need to use the assignment to `m` and can just do `requests_mock.call_count == 1` using the fixture directly.